src/components/Modals/OnboardingBasic.vue

```diff
@@ -0,0 +1,508 @@
+                  >
+                    <div
+                      v-for="(college, acronym) in colleges"
+                      :key="acronym"
+                      :id="college"
+                      class="onboarding-dropdown-content-item"
+                      @click="selectCollege(college, acronym, index)"
+                    >
+                      {{ college }}
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+
+            <div class="onboarding-inputWrapper onboarding-inputWrapper--college">
+              <label class="onboarding-label">Your Major!</label>
+              <div class="onboarding-selectWrapper">
+                <div
+                  class="onboarding-select onboarding-input"
+                  :class="{ 'onboarding-select--disabled': Object.keys(majors).length <= 0 }"
+                  id="major"
+                  v-for="(options, index) in displayOptions.major"
+                  :key = index
+                  :style="{ borderColor: options.boxBorder }"
+                  v-click-outside:[index]="closeMajorDropdownIfOpen"
+                >
+                  <div class="onboarding-dropdown-placeholder major-wrapper" @click="showHideMajorContent(index)">
+                    <div
+                      class="onboarding-dropdown-placeholder major-placeholder"
+                      id="major-placeholder"
+                      :style="{ color: options.placeholderColor }"
+                    >
+                      {{options.placeholder}}
+                    </div>
+                    <div
+                      class="onboarding-dropdown-placeholder major-arrow"
+                      id="major-arrow"
+                      :style="{ borderTopColor: options.arrowColor }"
+                    ></div>
+                  </div>
+                  <div
+                    class="onboarding-dropdown-content major-content"
+                    id="major-content"
+                    v-if="options.shown"
+                  >
+                    <div
+                      v-for="(major, acronym) in majors"
+                      :key="acronym"
+                      :id="major"
+                      class="onboarding-dropdown-content-item"
+                      @click="selectMajor(major, acronym, index)"
+                    >
+                      {{ major }}
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="onboarding-addRemoveWrapper" :class="{ 'onboarding--hidden': displayOptions.major.length <= 0}">
+                <div class="onboarding-add" @click="addMajor">
+                  Add
+                </div>
+                <div class="onboarding-remove" @click="removeMajor" :class="{ 'onboarding--hidden': displayOptions.major.length <= 1 }">
+                  Remove
+                </div>
+              </div>
+            </div>
+            <!--
```

Comment:
Is minor commented out because it was when you branched off master? If so - you should try merging master in since it is enabled on our live site now... Onboarding will get messy though.